src/router.ts

```diff
@@ -1415,6 +1424,16 @@ export async function handleClearedVoter(
       ...activeChannelMessageParams,
       isAutomatedMessage: true,
     });
+
+    if (process.env.CLIENT_ORGANIZATION === 'GADEMS') {
+      // GA Dems starts a new thread if an old voters comes back rather
+      // than continuing to use the existing thread. We do this by "routing"
+      // the voter back to the same channel
+      await routeVoterToSlackChannel(userInfo, redisClient, {
+        twilioPhoneNumber,
+        destinationSlackChannelName: userInfo.activeChannelName,
+      });
+    }
```

Comment:
This will make the voter pop up again in the channel, but it won't be a new "session" in that the voter history from the previous day(s) wont' be collapsed. That might be fine for their use-case though?

Reply:
yeah, i think this is desirable for ga dems (we're only looking at ~2 weeks total of interactions so i think it's fine for it all to be inline)